src/modules/my-orders/actions/update-orders.js

```diff
@@ -5,7 +5,7 @@ import logError from 'utils/log-error'
 import { BUY, SELL } from 'modules/transactions/constants/types'
 
 export const updateSingleMarketOrderBook = (updatedOrdersInMarket, isOrderCreation) => (dispatch, getState) => (
-  Object.keys(updatedOrdersInMarket).forEach(outcome => updatedOrdersInMarket[outcome].forEach(orderLog => (dispatch(updateOrderBook(orderLog.marketId, outcome, (orderLog.orderType === '0' ? BUY : SELL), { [orderLog.orderId]: orderLog })))))
+  Object.keys(updatedOrdersInMarket).forEach(outcome => updatedOrdersInMarket[outcome].forEach(orderLog => (dispatch(updateOrderBook(orderLog.marketId, outcome, (orderLog.orderType === 'buy' ? BUY : SELL), { [orderLog.orderId]: orderLog })))))
 )
```

Comment:
The values are not consistent:

handleOrderCanceledLog {"universe":"0xa0e4dc62cb7429825f6e1b49841a0445b37152bb","shareToken":"0xefe3cf86e381acee95b8cbcc0aec3216922c86e8","sender":"0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb","orderId":"0x84d5ce6715766954190e1728357d6aa7adc63fc0b6cd2a1cb26919cf1c0fc83f",**"orderType":"1"**,"tokenRefund":"1800000000000000","sharesRefund":"0","address":"0x25ff5dc79a7c4e34254ff0f4a19d69e491201dd3","removed":false,"transactionHash":"0xe08bd311d8c5215a0c2f027b7e0bbe9b133e4b6ce2a9db7446a29e1383f6b15b","transactionIndex":0,"logIndex":1,"blockNumber":900,"blockHash":"0x07655cd7b54f76c8862e71ec1415e50723ac2bf6b16985b0be0bf4a9a3f89d9e","contractName":"Augur","eventName":"OrderCanceled","marketId":"0x7d896c8803c98acc8cd5efd2c94d078e04eeb1f7","outcome":1,"price":"0.4"}
4clear-order-book-on-first-chunk.js?1bca:6 

handleOrderFilledLog {"universe":"0xa0e4dc62cb7429825f6e1b49841a0445b37152bb","shareToken":"0xefe3cf86e381acee95b8cbcc0aec3216922c86e8","filler":"0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb","orderId":"0x0689ad2d2e6216dd53a4ce3b10c8d958aac88ced22f97b98dcb6758b35727708","numCreatorShares":"0","numCreatorTokens":"0.00028","numFillerShares":"0","numFillerTokens":"0.00072","marketCreatorFees":"0","reporterFees":"0","amountFilled":"100000000000","tradeGroupId":"0xa7eba6321a14b8ddb5997dcfa50113f10bb6513a1a533bc2dbd8f38cc58f4668","address":"0x25ff5dc79a7c4e34254ff0f4a19d69e491201dd3","removed":false,"transactionHash":"0xea5113b2b214a46e754cfce3b0720e991f7901baaa272ceb1002f256e80d1ef1","transactionIndex":0,"logIndex":2,"blockNumber":883,"blockHash":"0x2f8ec3d814a4885295072adccd469a6ef522538630bc80ec8f045b4330713116","contractName":"Augur","eventName":"OrderFilled","marketId":"0x7d896c8803c98acc8cd5efd2c94d078e04eeb1f7","outcome":1,"creator":"0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb",**"orderType":"buy"**,"price":"0.28","amount":"0.001"}